### DISCUSSION TITLE
My issue #11997 is not fixed i think

### DISCUSSION THREAD

**Discussion by KINGER404** [quasarframework/quasar#12153] [General - Components / Directives / etc] [answered]
#11997 is not fixed
New reproduction URL: https://codepen.io/kinger404/pen/ZEXgELM?editors=1010

On first screen everything ok

<img width="300" alt="изображение" src="https://user-images.githubusercontent.com/53098109/150613973-0a6828e1-4df8-4ed9-b84e-68c72ebe87f1.png">
On second screen first q-step title missing

<img width="300" alt="изображение" src="https://user-images.githubusercontent.com/53098109/150614100-af5e1e31-db1f-46c5-9278-1c5466241e3a.png">

Step to reproduce describe in #11997

**Comment by pdanpdan** [comment]
What browser/os, because I cannot reproduce it

Neither in the new codepen you provided nor in the original one (updated to latest version)
https://codepen.io/pdanpdan/pen/jOGgEYj?editors=1010

**Reply by KINGER404** [reply]
I check this bug in Safari Chrome Firefox on masOs and Firefox Opera on windows

In firefox on masOs and windows bug saved, fixed in others.

**Reply by KINGER404** [reply]
And new bug is q-stepper-navigation are missing in QStep with QEditor by default.

**Comment by pdanpdan** [comment]
So you only have the problem in Firefox, and in other browsers it is fixed?
What version of Firefox?

**Reply by KINGER404** [reply]
Windows: 96.0.1
MasOs: 96.0.1
Other browsers fixed.

**Comment by pdanpdan** [comment, marked as answer]
> And new bug is q-stepper-navigation are missing in QStep with QEditor by default.

Don't use self closing tags in codepens (never when using UMD)
